### PR TITLE
feat: Exposes a method for checking is remote track received and played.

### PR DESCRIPTION
**Pull request by damencho**
Used for some tests in torture.

### CODE REVIEW

File: modules/UI/videolayout/VideoContainer.js [hunk 1]
```diff
@@ -259,6 +269,18 @@ export class VideoContainer extends LargeContainer {
...
         this.$video[0].onresize = this._onResize.bind(this);
+
+        if (isTestModeEnabled(APP.store.getState())) {
+            const cb = name => APP.store.dispatch(updateLastLargeVideoMediaEvent(name));
+            const containerHandlers = {};
+
+            containerEvents.forEach(event => {
+                containerHandlers[event] = cb.bind(this, event);
+            });
+            containerEvents.forEach(event => {
```
Comment (hristoterezov) [inline modules/UI/videolayout/VideoContainer.js:280]: I don't think you need a second loop here. You can add the code for adding a listener in the previous one, isn't it?

File: react/features/base/testing/middleware.js [hunk 2]
```diff
@@ -15,13 +18,19 @@ import logger from './logger';
...
         _bindConferenceConnectionListener(action.conference, store);
         break;
+    case SET_CONFIG:
+        result = next(action);
+        _bindTortureHelpers(store);
+        break;
     }
 
-    return next(action);
+    return result || next(action);
```
Comment (hristoterezov) [inline react/features/base/testing/middleware.js:33]: This looks simpler to me! WDYT?

```
  MiddlewareRegistry.register(store => next => action => {
    switch (action.type) {
    case CONFERENCE_WILL_JOIN:
        _bindConferenceConnectionListener(action.conference, store);
        break;
    case SET_CONFIG:
        const result = next(action);

        _bindTortureHelpers(store);
        
        return result;
    }

    return next(action);
```

File: lang/main.json [hunk 3]
```diff
@@ -393,8 +393,7 @@
         "toggleFilmstrip": "Show or hide video thumbnails",
         "toggleScreensharing": "Switch between camera and screen sharing",
         "toggleShortcuts": "Show or hide keyboard shortcuts",
-        "videoMute": "Start or stop your camera",
-        "videoQuality": "Manage call quality"
```
Comment (hristoterezov) [inline lang/main.json:397]: This doesn't seem related to the PR. Just wanted to check if you have accidentally committed it?

File: react/features/toolbox/components/web/Toolbox.js [hunk 4]
```diff
@@ -275,7 +275,7 @@ class Toolbox extends Component<Props, State> {
             this._shouldShowButton('videoquality') && {
                 character: 'A',
                 exec: this._onShortcutToggleVideoQuality,
-                helpDescription: 'keyboardShortcuts.videoQuality'
+                helpDescription: 'toolbar.callQuality'
```
Comment (hristoterezov) [inline react/features/toolbox/components/web/Toolbox.js:278]: This doesn't seem related to the PR. Just wanted to check if you have accidentally committed it?

Comment (hristoterezov) [reply react/features/toolbox/components/web/Toolbox.js:278]: My bad! Just saw it is separate commit so I guess it is not accidental :)

File: lang/main.json (hunk 3 above)
Comment (hristoterezov) [reply lang/main.json:397]: My bad! Just saw it is in separate commit so I guess it is not accidental :)

**Review by hristoterezov — CHANGES_REQUESTED** [review]
That's awesome!

Left some very minor comments!

File: modules/UI/videolayout/VideoContainer.js (hunk 1 above)
Comment (damencho) [reply modules/UI/videolayout/VideoContainer.js:280]: 🤦

File: react/features/base/testing/middleware.js (hunk 2 above)
Comment (damencho) [reply react/features/base/testing/middleware.js:33]: That does not make lint happy, no const in case clause, that's the reason I did it this way.

Comment (damencho) [reply react/features/base/testing/middleware.js:33]: `error  Unexpected lexical declaration in case block  no-case-declarations`

Comment (damencho) [reply react/features/base/testing/middleware.js:33]: 🤦

Comment (damencho) [reply react/features/base/testing/middleware.js]: The error was totally different, it is fixed now.

File: modules/UI/videolayout/VideoContainer.js (hunk 1 above)
Comment (damencho) [reply modules/UI/videolayout/VideoContainer.js]: 1:1 copy paste from lib-jitsi-meet 😉

**Review by hristoterezov — DISMISSED** [review]

**Review by hristoterezov — APPROVED** [review]
